contracts/MemeFactory.sol

```diff
@@ -419,26 +412,32 @@ abstract contract MemeFactory {
     }
 
     /// @dev Converts collected reference token to OLAS.
-    function scheduleOLASForAscendance(uint256 slippage) external virtual {
+    function scheduleOLASForAscendance(uint256 amount, uint256 slippage) external virtual {
```

Comment:
This amount is strictly needed here. Otherwise this can become unusable, requiring a flashloan again.